### ISSUE TITLE
published_publish subcommand options are not recognized

### ISSUE THREAD

**Issue by thoreg** [divio/django-cms#5171]
Hi,

following the documentation under http://docs.django-cms.org/en/3.2.3/reference/cli.html#cms-publisher-publish - I am trying to publish sites with a specific language.

The command: 
```
python manage.py cms publisher_publish --language=de
``` 
leads to the error message: 
```
manage.py: error: no such option: --language
```
The "help" of the subcommand (any subcommand?) seems not to work too:

```
python manage.py cms publisher_publish --help
Usage: manage.py cms [options] <subcommand>

django CMS command line interface.

Available subcommands:
  check
  copy-lang
  copy-site
  delete_orphaned_plugins
  fix-tree
  list
  moderator
  publisher_publish
  uninstall

Use `manage.py cms <subcommand> --help` for help about subcommands

Options:
  --version             show program's version number and exit
  -h, --help            show this help message and exit
  -v VERBOSITY, --verbosity=VERBOSITY
                        Verbosity level; 0=minimal output, 1=normal output,
                        2=verbose output, 3=very verbose output
  --settings=SETTINGS   The Python path to a settings module, e.g.
                        "myproject.settings.main". If this isn't provided, the
                        DJANGO_SETTINGS_MODULE environment variable will be
                        used.
  --pythonpath=PYTHONPATH
                        A directory to add to the Python path, e.g.
                        "/home/djangoprojects/myproject".
  --traceback           Raise on CommandError exceptions
  --no-color            Don't colorize the command output.
  --noinput             Tells django-cms to NOT prompt the user for input of
                        any kind.
```

I am using the following versions:
```
pip freeze | grep django-cms
django-cms==3.2.3

pip freeze | grep Django
Django==1.9.4
```

Do I do anything wrong?
 
Many thanks in advance

**Comment by yakky** [comment]
I edited the title to describe the root issue
Not sure how we can fix this: easiest fix is to move from options to arguments, like in `copy-site` / `copy-lang`
This is solved in 3.3 in the complete refactoring of the django CMS subcommands

**Comment by czpython** [comment]
@yakky Can we close this as it's fixed on 3.3?

**Comment by yakky** [comment]
@czpython yes
